packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -148,17 +167,33 @@ const init = ({ mainWindow, store }: Dependencies) => {
                     releaseDate,
                     downloadedFile,
                 });
+
+                shouldInstallUpdateOnQuit = true;
             })
             .catch(err => {
                 logger.error('auto-updater', `Signature check failed: ${err.message}`);
 
                 mainWindow.webContents.send('update/error', err);
 
+                logger.info('auto-updater', `Unlink downloaded file ${downloadedFile}`);
+
                 // Delete file so we are sure it's not accidentally installed
                 unlinkSync(downloadedFile);
+
+                shouldInstallUpdateOnQuit = false;
+            })
+            .finally(() => {
+                logger.info(
+                    'auto-updater',
+                    `Is configured to auto update after app quit ${shouldInstallUpdateOnQuit}`,
+                );
             });
     });
 
+    autoUpdater.on('before-quit-for-update', () => {
+        logger.info('auto-updater', 'before-quit-for-update event');
+    });
```

Comment:
Nitpick, optional: Personally, I'd omit this. If we want to get logs about what the `electron-updater` is doing then let's use its [dedicated api](https://github.com/trezor/trezor-suite/blob/b1e90d0712d37df9909dc380bceb19f7b1a259ac/packages/suite-desktop/src-electron/modules/auto-updater.ts#L52).

Reply:
RBY